build.gradle

```diff
@@ -163,11 +170,27 @@ subprojects {
 
     apply from: rootProject.file('gradle/teamcity.gradle')
     // Configure publishing for some artifacts
-    if (project.name != "benchmark" && project.name != "guide") {
+    if (project.name != "benchmark" && project.name != "guide" && !(project.name in multiplatformProjects)) {
```

Comment:
Why make this condition and move publishing setup to other gradle files?

Reply:
There is some problem at the moment: user-defined variant attributes (JVM compiler type in our case) are not taken into account when publishing is set up before applying kotlin-multiplatform plugin